Zend/zend_interfaces.stub.php

```diff
@@ -30,17 +30,30 @@ public function rewind();
 
 interface ArrayAccess
 {
-    /** @return bool */
+    /**
+     * @param mixed $offset
+     * @return bool
+     */
     public function offsetExists($offset);
```

Comment:
Is it not possible to add a real `mixed` type?

Reply:
You're right, we can add them!